lib/logthrdest/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(LOGTHRDEST_HEADERS
+    logthrdest/logthrdestdrv.h
+    logthrdest/logthrfetcherdrv.h
+    PARENT_SCOPE)
+
+set(LOGTHRDEST_SOURCES
+    logthrdest/logthrdestdrv.c
+    logthrdest/logthrfetcherdrv.c
```

Comment:
Oops, I did not see that `logthrfetcherdrv` was left here.
I've tested the CMake build on my laptop, this should be the last CMake-related failure.

Reply:
thanks. fixed this one as well.